dlt/destinations/impl/filesystem/filesystem.py

```diff
@@ -171,27 +179,44 @@ def update_stored_schema(
         self, only_tables: Iterable[str] = None, expected_update: TSchemaTables = None
     ) -> TSchemaTables:
         # create destination dirs for all tables
-        dirs_to_create = self._get_table_dirs(only_tables or self.schema.tables.keys())
-        for directory in dirs_to_create:
+        table_names = only_tables or self.schema.tables.keys()
+        dirs_to_create = self._get_table_dirs(table_names)
+        for tables_name, directory in zip(table_names, dirs_to_create):
             self.fs_client.makedirs(directory, exist_ok=True)
+            # we need to mark the folders of the data tables as initialized
+            if tables_name in self.schema.dlt_table_names():
+                print(directory + " " + tables_name)
+                self.fs_client.touch(f"{directory}/init")
+
+        # write schema to destination
+        self.store_current_schema()
+
         return expected_update
 
-    def _get_table_dirs(self, table_names: Iterable[str]) -> Set[str]:
+    def _get_table_dirs(self, table_names: Iterable[str]) -> List[str]:
         """Gets unique directories where table data is stored."""
-        table_dirs: Set[str] = set()
+        table_dirs: List[str] = []
```

Comment:
so that the def update_stored_schema for the filesystem works, for that I need an ordered list of dirs. I can also change it so we keep the set here and implement another one for the the schema update.